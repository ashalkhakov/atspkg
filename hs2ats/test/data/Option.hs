module Option where

data Option a = Some a
              | None
