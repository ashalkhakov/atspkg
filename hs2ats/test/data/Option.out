datavtype option(a: vt@ype) =
  | Some of a
  | None
